### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by candyhy**
Update StorageClassDiagram.puml
Update Storage component of Developer's guide

### CODE REVIEW

**Comment by candyhy** [comment]
Updated StorageClassDiagram
StorageClassDiagram

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -129,11 +129,12 @@ The `Model`,
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2021S2-CS2103T-T13-4/tp/blob/master/src/main/java/seedu/address/storage/Storage.java)
 
 The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
```
Comment (w2vgd) [inline docs/DeveloperGuide.md]: Should this be `UserPrefs` instead?

**Review by w2vgd — COMMENTED** [review]
A small typo

**Comment by w2vgd** [comment]
> StorageClassDiagram

`Storage` interface should extend `PropertyBookStorage`, `AppointmentBookStorage` and `UserPrefsStorage` too

image

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (candyhy) [reply docs/DeveloperGuide.md]: oh yes ok will update this

**Comment by candyhy** [comment]
> > StorageClassDiagram
> 
> `Storage` interface should extend `PropertyBookStorage`, `AppointmentBookStorage` and `UserPrefsStorage` too
> 
> image

will update this too

**Comment by candyhy** [comment]
new `StorageClassDiagram`
StorageClassDiagram

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**